boards/st/stm32n6570_dk/stm32n6570_dk_common.dtsi

```diff
@@ -46,6 +46,21 @@
 			label = "User LD2";
 		};
 	};
+
+	csi_camera_connector: connector_csi_camera {
+		compatible = "st,stm32-csi-camera";
```

Comment:
It seems like [this camera module](https://www.st.com/en/evaluation-tools/b-cams-imx.html?ecmp=tt9470_gl_link_feb2019&rt=db&id=DB5351) is using the de-facto standard 22-pin connector introduced by Raspberry Pi.

It might be possible to define an extra alias label i.e. at the bottom of the `.dts`:
https://github.com/zephyrproject-rtos/zephyr/blob/4a31b8036d9c20349a0ee9ab83fa86147fdf983e/boards/nxp/mimxrt1064_evk/mimxrt1064_evk.dts#L353-L356

So no the label used inline can be something ST-specific if that makes it more convenient.

In hope this helps shaping-up a shield:

- https://github.com/zephyrproject-rtos/zephyr/pull/89131